### PR TITLE
check if the keys directory exists have to be delegated

### PR DESCRIPTION
**Pull request by muellerbe**
Signed-off-by: Bernd Mueller <mueller@b1-systems.de>

and fileglob needs /* at the end

### CODE REVIEW

File: roles/repository/tasks/repository-Debian.yml [hunk 1]
```diff
@@ -18,13 +18,14 @@
   ansible.builtin.stat:
     path: "{{ repository_key_files_directory }}"
   register: result
+  delegate_to: "{{ groups['manager'][0]"
```
Comment (osfrickler) [inline roles/repository/tasks/repository-Debian.yml:21]: Missing closing }}

Comment (muellerbe) [reply roles/repository/tasks/repository-Debian.yml]: thank you

File: roles/repository/tasks/repository-Debian.yml [hunk 2]
```diff
@@ -18,13 +18,14 @@
   ansible.builtin.stat:
     path: "{{ repository_key_files_directory }}"
   register: result
+  delegate_to: "{{ groups['manager'][0] }}"
```
Comment (berendt) [inline roles/repository/tasks/repository-Debian.yml:21]: I trink it is better to delegate this one to localhost.

Comment (muellerbe) [reply roles/repository/tasks/repository-Debian.yml]: yes, done in new commit

**Pull request merged**